pages/common/atoum.md

```diff
@@ -0,0 +1,36 @@
+# atoum
+
+> A simple, modern and intuitive unit testing framework for PHP.
+> Homepage: <https://atoum.org>.
+
+- Initialise a configuration file:
+
+`atoum --init`
+
+- Run all tests:
+
+`atoum`
+
+- Run tests using the specified configuration file:
+
+`atoum -c {{path/to/configuration_file}}`
```

Comment:
Did you intentionally want to write configuration_file and bootstrap_file, and just file for the test file ? I think path/to/file in all cases should be fine because the description already says what kind of file it is.

Reply:
I think I wrote it to avoid confusion, but then I updated the examples before committing and forgot to change it.